rustworkx/__init__.py

```diff
@@ -1171,6 +1171,10 @@ def closeness_centrality(graph, wf_improved=True):
         :class:`~rustworkx.PyGraph` or :class:`~rustworkx.PyDiGraph`.
     :param bool wf_improved: This is optional; the default is True. If True,
         scale by the fraction of nodes reachable.
+    :param int parallel_threshold: The number of nodes to calculate the
+        the closeness centrality in parallel at if the number of nodes in
+        the graph is less than this value it will run in a single thread. The
+        default value is 50.
```

Comment:
Do you want to mention `RAYON_NUM_THREADS` to control the size of the thread pool here too? We do this in the other parallel functions typically.

Reply:
Done.